scripts/go_install_binaries.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1
+go install golang.stackrox.io/kube-linter/cmd/kube-linter@v0.0.0-20210328011908-cb34f2cc447f
+go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.43.0
```

Comment:
This installation method is not recommended:
https://golangci-lint.run/usage/install/#install-from-source
> Note: such go get installation aren't guaranteed to work. We recommend using binary installation

Maybe this is connected to the intermittent issues with `make lint` and should we use the recommended method of installation?